pkg/storaged/dialog.jsx

```diff
@@ -1268,15 +1269,16 @@ export const StopProcessesMessage = ({ mount_point, users }) => {
                                       ]
                                   }
                                       rows={process_rows} />
-                </p>
+                </>
                 : null
             }
             { process_rows.length > 0 && service_rows.length > 0
                 ? <br />
                 : null
             }
             { service_rows.length > 0
-                ? <p>{fmt_to_fragments(_("The mount point $0 is in use by these services:"), <b>{mount_point}</b>)}
+                ? <>
+                    <p>{fmt_to_fragments(_("The mount point $0 is in use by these services:"), <b>{mount_point}</b>)}</p>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231024-102436-6e7b9faf-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)